### PR TITLE
Pac update

### PR DESCRIPTION
**Pull request by diagonalDave**
This PR has regenerated the PAC, provided a generate.bat and gated interrupts for the CM0+ core.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,38 +1,43 @@
...
 name = "psoc6-pac"
-version = "0.0.0"
-authors = ["Jonas Schievink <jonasschievink@gmail.com>"]
+version = "0.0.1"
+authors = ["Jonas Schievink <jonasschievink@gmail.com>", "Dave Humphries <humphriesdj@gmail.com"]
 edition = "2018"
 description = "Peripheral Access Crate for Cypress PSoC 6 MCUs"
 documentation = "https://docs.rs/psoc6-pac/"
-repository = "https://github.com/jonas-schievink/psoc6-pac.git"
+repository = "https://github.com/diagonalDave/psoc6-pac.git"
```
Comment (mvertescher) [inline Cargo.toml]: We can probably just point at `https://github.com/psoc-rs/psoc6-pac` here and use the `psoc-rs` org link for other spots too.

File: README.md [hunk 2]
```diff
@@ -18,11 +18,14 @@ Add an entry to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-psoc6-pac = "0.0.0"
+psoc6-pac = "0.1.1"
```
Comment (mvertescher) [inline README.md]: This should be `0.0.1` right?

**Review by mvertescher — CHANGES_REQUESTED** [review]
Thanks for posting this update @diagonalDave! Looks pretty good to me, just a few minor comments.

**Comment by diagonalDave** [comment]
Yep.

File: Cargo.toml (hunk 1 above)
Comment (diagonalDave) [reply Cargo.toml]: Hopefully resolved both issues in latest commit.

**Comment by mvertescher** [comment]
Thanks for the changes @diagonalDave LGTM!

**Pull request merged**